From 87be3e27ccc8f9a1ef7fbeec9600459eb4569200 Mon Sep 17 00:00:00 2001
From: Bobby Rong <rjl931189261@126.com>
Date: Mon, 14 Mar 2022 23:36:21 +0800
Subject: [PATCH] Fix build with vala 0.56

Otherwise build fails with

../src/Application.vala:45.50-55.5: error: value is less accessible than constant `Planner.PLANNER_OPTIONS'
---
 src/Application.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Application.vala b/src/Application.vala
index 8c1c9ddb..ebd8f726 100644
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -42,7 +42,7 @@ public class Planner : Gtk.Application {
     private static bool clear_database = false;
     private static string lang = "";
 
-    public const OptionEntry[] PLANNER_OPTIONS = {
+    private const OptionEntry[] PLANNER_OPTIONS = {
         { "version", 'v', 0, OptionArg.NONE, ref version,
         "Display version number", null },
         { "reset", 'r', 0, OptionArg.NONE, ref clear_database,
